modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractScalaCodegen.java

```diff
@@ -105,11 +102,27 @@ public AbstractScalaCodegen() {
                 "yield"
         ));
 
+        importMapping = new HashMap<String, String>();
         importMapping.put("ListBuffer", "scala.collection.mutable.ListBuffer");
         // although Seq is a predef, before Scala 2.13, it _could_ refer to a mutable Seq in some cases.
         importMapping.put("Seq", "scala.collection.immutable.Seq");
         importMapping.put("Set", "scala.collection.immutable.Set");
         importMapping.put("ListSet", "scala.collection.immutable.ListSet");
+        // fallback to java types
+        importMapping.put("UUID", "java.util.UUID");
+        importMapping.put("URI", "java.net.URI");
+        importMapping.put("File", "java.io.File");
+        importMapping.put("Timestamp", "java.sql.Timestamp");
+        importMapping.put("HashMap", "java.util.HashMap");
+        importMapping.put("Array", "java.util.List");
+        importMapping.put("ArrayList", "java.util.ArrayList");
+        // todo remove legacy date types
+        importMapping.put("Date", "java.util.Date");
+        importMapping.put("DateTime", "org.joda.time.*");
```

Comment:
For this and others, we should use java.time rather than Joda. Non-standard libraries should be optional.

Reply:
i've copied that from `DefaultCodegen` for backward compatibility.
Next PR https://github.com/OpenAPITools/openapi-generator/pull/5291 should solve java.time which should become default since 5.0.0, i think.